src/styles/blocks/card.scss

```
@@ -0,0 +1,96 @@
+.card {
+  width: 200px;
+  padding: 16px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  color: $main-text-color;
+
+  &__image {
+    margin-top: 16px;
+    margin-bottom: 40px;
+    margin-left: 3px;
+    width: 160px;
+    height: 134px;
+    background-size: 100% 100%;
+    background-position: top center;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin: 0;
```

need to set all margin from one side top or bottom dont mix it up
```suggestion
    margin: 40px 0 0;
```